spec/support/shared_contexts/rake.rb

```diff
@@ -0,0 +1,19 @@
+require "rake"
+
+shared_context "rake" do
+  let(:rake)      { Rake::Application.new }
+  let(:task_name) { self.class.top_level_description }
+  let(:task_path) { "lib/tasks/#{task_name.split(":").first}" }
+  subject(:task)         { rake[task_name] }
+
+  def loaded_files_excluding_current_rake_file
+    $".reject {|file| file == Rails.root.join("#{task_path}.rake").to_s }
```

Comment:
Minor style tweak (for our as-yet unwritten style guide, so my bad here too): please put a space before the block arguments. Thanks!